location.go

```diff
@@ -38,26 +43,61 @@ type Section struct {
 	Status        string          `json:"status"`
 }
 
-func (l *LocationInstance) Feeds(locationID int64) (*Section, error) {
-	// TODO: use pagination for location feeds.
-	insta := l.inst
+// Next allows to paginate after calling:
+// Locations.Feeds()
+// returns false when list reach the end.
+func (section *Section) Next() bool {
+	if section.err != nil {
+		return false
+	}
+
```

Comment:
We can paginate only if `MoreAvailable` is true. So please write an if condition instead of checking errors.